src/main/webapp/query-filters/filter-input/location-filter.tsx

```diff
@@ -3,17 +3,35 @@ import { QueryFilterProps } from '../filter/individual-filter'
 import { makeDefaultSearchGeo } from '../filter'
 import { Location, geoToFilter } from '../../location'
 import AttributeDropdown from '../filter/attribute-dropdown'
+const wkx = require('wkx')
+import { geoJSONToGeometryJSON } from 'geospatialdraw/bin/geometry/utilities'
+import { makeSearchGeoId } from '../filter/search-geo-factory'
+
+const getGeojson = (filter: QueryFilterProps['filter']) => {
+  if (filter.geojson) return filter.geojson
+
+  if (filter.value) {
+    let geojson = wkx.Geometry.parse(filter.value).toGeoJSON()
```

Comment:
is there a better way to convert a wkt string to geojson object without needing to bring in a new dependency?